pkg/operator/operator.go

```diff
@@ -63,10 +63,9 @@ import (
 	"sigs.k8s.io/karpenter/pkg/utils/env"
 )
 
-const (
-	appName   = "karpenter"
-	component = "controller"
-)
+const component = "controller"
```

Comment:
nit: I wonder if we can drop the component constant entirely since we only use it in one spot